docs/getting-started/installation.md

```diff
@@ -10,19 +10,16 @@ In this section you will find an aggregation of the different ways to install Tr
     Add repository setting to `/etc/yum.repos.d`.
 
     ``` bash
-    RELEASE_VERSION=$(grep -Po '(?<=VERSION_ID=")[0-9]' /etc/os-release)
-    cat << EOF | sudo tee -a /etc/yum.repos.d/trivy.repo
+    sudo tee /etc/yum.repos.d/trivy.repo << 'EOF'
     [trivy]
     name=Trivy repository
-    baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$RELEASE_VERSION/\$basearch/
+    baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$releasever/$basearch/
     gpgcheck=1
     enabled=1
     gpgkey=https://aquasecurity.github.io/trivy-repo/rpm/public.key
     EOF
-    sudo yum -y update
-    sudo yum -y install trivy
```

Comment:
I'm not sure we should replace `yum` with `dnf`, because `yum` is still default PM on Centos 7, isn't it?
maybe we add a new command. @Wang-W-e-i wdyt?